bin/live/pycbc_live_combine_single_significance_fits

```diff
@@ -62,8 +62,10 @@ for f in args.trfits_files:
             bl = fit_f['bins_lower'][:]
             bu = fit_f['bins_upper'][:]
             sngl_rank = fit_f.attrs['sngl_ranking']
-            fit_thresh = fit_f.attrs['fit_threshold']
-            fit_func = fit_f.attrs['fit_function']
+            fit_thresh = {ifo: fit_f[ifo].attrs['fit_threshold']
```

Comment:
Is it planned to actually use different values of `fit_func` and `fit_thresh` for different ifos in production?

Reply:
`fit_func` - probably not, `fit_thresh` - possibly

The sensitive distance thing shifts the distribution of the noise events by the log of volume, so a factor of 1.1 in distance becomes a change of 1.33 in the statistic.

As the event rate at a given stat is distributed as an exponential, this will be a factor of ~3.8 is the number of events, which when dealing with low-number statistics as we are, can be important.

Its fairly easy to see this in the offline results, if you look at A18/A19 in results pages

In the offline configs we use a difference of 1.2 in threshold in the detectors, so a factor of 3.3 in the number of events